resources/prosody-plugins/mod_jitsi_session.lua

```diff
@@ -18,7 +19,7 @@ function init_session(event)
         session.previd = query and params.previd or nil;
 
         -- The room name and optional prefix from the web query
-        session.jitsi_web_query_room = params.room;
+        session.jitsi_web_query_room = urlencode(params.room);
         session.jitsi_web_query_prefix = params.prefix or "";
```

Comment:
I think, we need to encode and the prefix if it is available.
This is the tenant ... so you can have meetings like https://example.jitsi.orig/täst/täst-room

Reply:
That I suspect is still broken, even after this PR?

Reply:
Thank you for the suggestion! I did that:

    session.jitsi_web_query_prefix = urlencode(params.prefix) or "";

but unfortunately tenants with special characters are still broken when testing with `/täst/test` (just to differentiate where the special char is):

```
Dec 07 12:24:27 c2s556f36c25dc0	debug	Received[c2s]: <message to='breakout.jitsi.example.org'>
Dec 07 12:24:27 jitsi.example.org:muc_breakout_rooms	warn	No room found found for /test
Dec 07 12:24:27 stanzarouter	debug	Unhandled c2s stanza: message; xmlns=jabber:client
Dec 07 12:24:27 c2s556f36c25dc0	debug	Sending[c2s]: <message from='breakout.jitsi.example.org' to='atmodgaopcw2jfsm@jitsi.example.org/9TjwWlV2' type='error'>
```

Any further suggestions?

Reply:
With another round of testing I noticed that av-moderation and breakouts are not working for any tenants and normal or special char rooms in my setup, so this may be an unrelated local issue.